src/nidigital/templates/session.py/fancy_get_pin_results_pin_information.py.mako

```diff
@@ -9,9 +9,11 @@
         ${helper.get_function_docstring(f, False, config, indent=8)}
         '''
         import collections
-        PinInfo = collections.namedtuple('PinInformation', ['pin_indexes', 'site_numbers', 'channel_indexes'])
+        PinInfo = collections.namedtuple('PinInformation', ['pin_name', 'site_number', 'channel_name'])
 
         pin_indexes, site_numbers, channel_indexes = self._${f['python_name']}()
+        assert len(pin_indexes) == len(site_numbers), "length of returned arrays don't match"
+        assert len(pin_indexes) == len(channel_indexes), "length of returned arrays don't match"
 
-        return [PinInfo(pin_indexes=pin_indexes[i], site_numbers=site_numbers[i], channel_indexes=channel_indexes[i]) for i in range(len(pin_indexes))]
+        return [PinInfo(pin_name=self.get_pin_name(pin_indexes[i]), site_number=site_numbers[i], channel_name=self.get_channel_name(channel_indexes[i])) for i in range(len(pin_indexes))]
```

Comment:
I'm getting an error when I run my system test:

```
>   return [PinInfo(pin_name=self.get_pin_name(pin_indexes[i]), site_number=site_numbers[i], channel_name=self.get_channel_name(channel_indexes[i])) for i in range(len(pin_indexes))]
E   AttributeError: '_SessionBase' object has no attribute 'get_pin_name'
```

Reply:
Moved `get_pin_name()` and `get_channel_name()` to `_SessionBase` since they are needed by `get_pin_results_pin_information()`